docs/sources/http_api/serviceaccount.md

```diff
@@ -0,0 +1,341 @@
++++
+title = "service account HTTP API "
+description = "Grafana service account HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "serviceaccount"]
+aliases = ["/docs/grafana/latest/http_api/serviceaccount/"]
++++
+
+# Service account API
+
+> If you are running Grafana Enterprise and have [Fine-grained access control]({{< relref "../enterprise/access-control/_index.md" >}}) enabled, for some endpoints you would need to have relevant permissions.
+> Refer to specific resources to understand what permissions are required.
+
+## Search service accounts with Paging
+
+`GET /api/serviceaccounts/search?perpage=10&page=1&query=myserviceaccount`
+
+#### Required permissions
+
+See note in the [introduction]({{< ref "#user-api" >}}) for an explanation.
+
+| Action               | Scope                     |
+| -------------------- | ------------------------- |
+| serviceaccounts:read | global:serviceaccounts:\* |
+
+**Example Request**:
+
+```http
+GET /api/serviceaccounts/search?perpage=10&page=1&query=mygraf HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Basic YWRtaW46YWRtaW4=
+```
+
+Default value for the `perpage` parameter is `1000` and for the `page` parameter is `1`. The `totalCount` field in the response can be used for pagination of the user list E.g. if `totalCount` is equal to 100 users and the `perpage` parameter is set to 10 then there are 10 pages of users. The `query` parameter is optional and it will return results where the query value is contained in one of the `name`. Query values with spaces need to be URL encoded e.g. `query=Jane%20Doe`.
```

Comment:
I feel like this content could go into a table to enhance readability.

| Parameter               | Description                     |
| -------------------- | ------------------------- |
| `perpage` | The default value is `1000` and add rest of text |
| `total count` | add description |

etc.